src/cli/near.rs

```diff
@@ -122,6 +182,89 @@ pub enum WriteCommand {
     FactoryUpdate {
         wasm_bytes_path: String,
     },
+    // deploy_code
+    DeployCode {
+        code_byte_hex: String,
+    },
+    // call
+    Call {
+        call_byte_hex: String,
+    },
+    // register_relayer
+    RegisterRelayer {
+        relayer_eth_address_hex: String,
+    },
+    // ft_on_transfer
+    FTOnTransfer {
+        sender_near_id: String,
+        amount: String,
+        msg: String,
+    },
```

Comment:
Note: with the split of the NEP-141 logic from the engine, all these `ft_*` commands will be deprecated.

Reply:
ok